prov/efa/src/rdm/efa_rdm_pke_rtm.c

```diff
@@ -1172,11 +1172,11 @@ ssize_t efa_rdm_pke_init_longread_tagrtm(struct efa_rdm_pke *pkt_entry,
  */
 void efa_rdm_pke_handle_longread_rtm_sent(struct efa_rdm_pke *pkt_entry)
 {
-	struct efa_rdm_peer *peer;
+	struct efa_domain *domain;
 
-	peer = efa_rdm_ep_get_peer(pkt_entry->ep, pkt_entry->addr);
-	assert(peer);
-	peer->num_read_msg_in_flight += 1;
+	domain = efa_rdm_ep_domain(pkt_entry->ep);
+	assert(domain);
```

Comment:
nit: I think the `assert(domain)` are not necessary because we cannot have endpoint without a domain

Reply:
Done